

class OfflineErr(Exception):
    pass

class NetWorkErr(Exception):
    pass
